### DISCUSSION TITLE
How to start up in Windows natively?

### DISCUSSION THREAD

**Discussion by jackyluo-learning** [serverlessworkflow/synapse#322] [Q&A] [answered]
Hey guys! I have installed using command "synctl system install native", and then navigate to "http://localhost:42286", but get "ERR_CONNECTION_REFUSED" message. Is that means the application has not start up?

**Comment by JBBianchi** [comment, marked as answer]
Hello @jackyluo-learning 

`synctl system install native` only installs Synapse (it downloads the binaries to `C:\Program Files\CNCF\Synapse\` and adds that directory to the `%PATH%`). You'll still have to run `synapse`. If you restart your term (powershell or cmd) session, you'll be able to run that command from anywhere.

(Note: it also installs `jq` in `C:\Program Files\jq\`)

The first time Synapse runs, it will download its worker in `C:\Program Files\CNCF\Synapse\bin\worker`. This directory requires admin rights to be written in.

@cdavernas It might be a good idea to allow to choose the installation directory to avoid having to run synctl\synapse as admin. An `uninstall` command would be nice too.

@jackyluo-learning an alternative if you'd like to avoid running the programs as admin would be to download synapse server archive ([synapse-win-x64.zip](https://github.com/serverlessworkflow/synapse/releases/download/v0.3.14/synapse-win-x64.zip)), extract it wherever you'd wish, optionally add the directory to your %PATH% for ease of use and run synapse.

If you like Synapse, please consider dropping a star, that means a lot to us <3
